modules/Emails/include/ListView/ListViewDataEmails.php

```diff
@@ -549,20 +549,20 @@ protected function getEmailRecordFieldValue($field, $emailHeader, $inboundEmail,
                 $importedEmailBeans = BeanFactory::getBean('Emails');
                 $is_imported = $importedEmailBeans->get_full_list(
                     '',
-                    'emails.uid LIKE "' . $uid . '"'
+                    'emails.uid LIKE "' . $uid . '" AND emails.mailbox_id = "' . $inboundEmail->id . '"'
```

Comment:
Please use quote() or quoted() method of DBManager before concat into a query string.